src/source/PeerConnection/PeerConnection.h

```diff
@@ -114,10 +124,10 @@ typedef struct {
     // congestion control
     // https://tools.ietf.org/html/draft-holmer-rmcat-transport-wide-cc-extensions-01
     UINT16 twccExtId;
-    UINT64 onPacketNotReceivedCustomData;
-    UINT64 onPacketReceivedCustomData;
-    RtcpCCOnPacketNotReceived onPacketNotReceived;
-    RtcpCCOnPacketReceived onPacketReceived;
+    MUTEX twccLock;
+    TwccManager twccManager;
```

Comment:
This is a little concerning because TwccManager  object would be about 1.2MB in size even with great cache locality. It would be a concern in a heavily memory constricted device. This is what I am thinking: 

We can have an option (through CMake option or a #define) to enable TWCC altogether. If enabled, we can allocate memory for `twccManager` and use it. Essentially, I think we should replace TwccManager with a pointer to the struct and then allocate memory if TWCC needs to be used. This would ideally bring back `#define ENABLE_TWCC` if we go ahead with the compile time switch. What do you think?